packages/template-retail-react-app/app/pages/social-login-redirect/index.jsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {Fragment} from 'react'
+import {FormattedMessage} from 'react-intl'
+import {Box, Container, Stack, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {BrandLogo} from '@salesforce/retail-react-app/app/components/icons'
+
+const SocialLoginRedirect = () => {
+    return (
+        <Box data-testid="login-redirect-page" bg="gray.50" py={[8, 16]}>
+            <Container
+                paddingTop={16}
+                width={['100%', '407px']}
```

Comment:
`407px` 🤔

Reply:
The registration, login, and reset password pages are using this width size so I copied this over for consistency😅
Example [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/template-retail-react-app/app/pages/reset-password/index.jsx#L64)